content/ja/docs/getting-started/dev.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 開発者のための入門
+linkTitle: Dev
+default_lang_commit: 93ab1301
+---
+
+あなたが以下のいずれかに当てはまれば、このページはあなたのための[入門](..)ページです。
+
+- ソフトウェアを開発している
+- あなたの目的は、コードを書くことでオブザーバビリティを得ることだ
+- 依存関係が自動的にテレメトリーを発するようにしたい
+
+OpenTelemetryが役に立ちます！
+依存関係を自動的に計装し、あなたのコードを手動で私たちのAPIで計装するというあなたの目標を達成するために、私たちはまず、次の概念を学ぶことをおすすめします。
+
+- [OpenTelemetryとは](/docs/what-is-opentelemetry/)
+- [コードベースに計装を追加するには](/docs/concepts/instrumentation/code-based/)
+
+他のソフトウェアで依存関係として使用されるライブラリ、フレームワーク、ミドルウェアを開発する場合、ネイティブにテレメトリーを提供する方法を学ぶことをおすすめします。
+
+- [自分のライブラリにネイティブ計装を追加するには](/docs/concepts/instrumentation/libraries/)
+
+次に、あなたが使っている[言語](/docs/languages/)のドキュメントを読んでください。
+
+- [C++](/docs/languages/cpp/)
+- [.NET](/docs/languages/net/)
+- [Erlang / Elixir](/docs/languages/erlang/)
+- [Go](/docs/languages/go/)
+- [Java](/docs/languages/java/)
+- [JavaScript / TypeScript](/docs/languages/js/)
+- [PHP](/docs/languages/php/)
+- [Python](/docs/languages/python/)
+- [Ruby](/docs/languages/ruby/)
+- [Rust](/docs/languages/rust/)
+- [Swift](/docs/languages/swift/)
+- [Other](/docs/languages/other/)
```

Comment:
For this page and the others too, ensure that you apply the changes made via:

- #4627

Reply:
I see. I change them to point to ja pages.

Reply:
I'd rather that you use relative paths as is done in the `en` page.

Reply:
My understanding is that #4627 was merged 2 days ago, so if I rebase onto `main` yesterday, the change was applied. Is it correct?

Anyway, I misunderstood your comment so I reverted the path change.

Reply:
No worries @ymotongpoo. Merging of translation PRs is on hold. I'll get back to you as soon as I can.